src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -41,7 +41,8 @@
     @JoinColumn(name = "CART_ID")
     private Cart cart;
 
-    public User(String name, String surname) {
+    public User(Long id, String name, String surname) {
```

Comment:
Wydaje mi się, że konstruktor bez ID też się przyda. Tworząc nowego użytkownika możemy nie chcieć podawać samemu ID, ale pozwolić bazie danych na wygenerowanie.

Reply:
Z tego co sprawdzałem POSTMANEM bez tego konstruktora i bez podania id tworzy sie nowy User i dodaje do bazy

Reply:
No pewnie w mapperze jest to mapowane poprawnie z użyciem bez parametrowego konstruktora i ustawiane name i surname osobno. To niech tak zostanie